services/arv-git-httpd/git_handler_test.go

```diff
@@ -41,13 +41,14 @@ func (s *GitHandlerSuite) TestEnvVars(c *check.C) {
 }
 
 func (s *GitHandlerSuite) TestCGIError(c *check.C) {
+	log.Printf("git_handler_test: TestCGIError() - expecting a SplitHostPort error, so it is safe to ignore!")
```

Comment:
Ok, happy with changing the name of the test and the bogus value to make it clear that a failure is expected.